ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -70,6 +70,78 @@ tests:
           cpu: "4"
           memory: 4Gi
     workflow: ipi-gcp
+- as: win10-pipeline-example-test
+  run_if_changed: (^automation/.*|^templates-pipelines/windows-bios-installer/.*|^templates-pipelines/windows-customize/.*|^modules/.*|^templates/.*)$
+  steps:
+    cluster_profile: gcp-virtualization
+    env:
+      COMPUTE_NODE_TYPE: n2-standard-4
+    test:
+    - as: win10-pipeline-example-test
+      commands: TARGET=windows10 ./automation/e2e-pipelines/example-pipelines-test.sh
+      credentials:
+      - mount_path: /tmp/secrets
+        name: common-templates-container-disk-puller
+        namespace: test-credentials
```

Comment:
This is secret, which we are using in e2e tests to download all windows / rhel images from private registry.
I can't rename it because the secret was originally created for common templates and we are reusing it in multiple projects. I would have to create another one with different name, but the same secret.

Reply:
Thank you.